pkgs/os-specific/darwin/iproute2mac/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, darwin, python }:
+{ stdenv, fetchFromGitHub, darwin, python2 }:
```

Comment:
we should open a ticket upstream about lack of python3 support.

Reply:
Yeah, but it's not a big deal since this isn't used anywhere. Just wanted to remove the last `python` reference in os-specific. 😄

Reply:
Actually, looks like there's already a pr for it.